iiif-pipeline.py

```diff
@@ -0,0 +1,18 @@
+import argparse
+import logging
+import os
+import shortuuid
+from configparser import ConfigParser
```

Comment:
This is unused; I know we talked about adding an issue for automated linting but it might be worth running flake8 over these to catch these issues.